pettingzoo/utils/env.py

```diff
@@ -180,57 +185,65 @@ def agent_iter(self, max_iter: int = 2**63) -> AECIterable:
         """
         return AECIterable(self, max_iter)
 
-    def last(self, observe: bool = True) -> Tuple[ObsType, float, bool, Dict[str, Any]]:
+    def last(
+        self, observe: bool = True
+    ) -> Tuple[ObsType, float, bool, bool, Dict[str, Any]]:
         """
-        Returns observation, cumulative reward, done, info   for the current agent (specified by self.agent_selection)
+        Returns observation, cumulative reward, terminated, info   for the current agent (specified by self.agent_selection)
```

Comment:
Should this comment say `termination, truncated`?

Reply:
done :)